en/docs/apis/api-concepts.md

```diff
@@ -1,23 +1,71 @@
+
+In Choreo, tokens are bound to an application. The Key manager provides a token API to generate access tokens. Clients can use these tokens to invoke APIs exposed by Choreo. The Key Manager also exposes a revoke token API that clients can use to revoke an access token. A client can generate an OAuth2.0 access token by invoking the token API directly or via the Developer Portal UI. Alternatively, a client can generate an API Key through the Developer Portal without calling the Key Manager. The API Key is a self-signed JWT token. When a client invokes an API with an OAuth2.0 access token or an API Key, the Gateway validates the token by validating its signature and subscription.
+
+### Choreo connect
+
+Choreo Connect is an API gateway designed for microservices that is cloud-native, decentralized, and developer-centric. This API gateway is a lightweight message processor for APIs that facilitates message security, transport security, routing, and other API Management-related quality of services such as throttling, rate-limiting, etc.
+
+Choreo Connect consists of three components: Router, Enforcer, and Adapter.
+
+![Choreo Connect](../assets/img/apis/choreo-connect.png){:style="height:75%;width:90%"}
+
+#### Router
+
+The Router in Choreo Connect uses the [Envoy Proxy](https://www.envoyproxy.io/) and is primarily responsible for routing traffic to the relevant endpoints. The Router is also capable of exposing the APIs in Choreo to make them discoverable.
+
+#### Enforcer
+
+The Enforcer is responsible for carrying out API management-related quality-of-services such as controlling access, enabling security, publishing analytics, etc. When the Router receives a request, it sends it to the Enforcer. The Enforcer performs the relevant API management-related quality of services on the request and replies to the Router. Based on the Router's response, the Enforcer will decide to allow or deny the request to pass to the backend.
+
+#### Adapter 
+
+The Adapter is responsible for converting the API definition to the respective format understood by the Router and the Enforcer. The Adapter will pass the data to and from the Router and Enforcer.
+
+### Traffic manager 
+
+The Traffic Manager helps regulate API traffic, makes APIs and applications available to consumers at different service levels, and secures APIs against security attacks. The Traffic Manager features a dynamic throttling engine that processes rate-limiting policies in real-time. 
+
+Additionally,  the Traffic Manager keeps the API Gateway's in-memory map, which it uses for key validation, up-to-date via a JMS topic. The Traffic Manager publishes artifact (API/application) update events that it receives from the API Publisher and API Developer Portal to a JMS topic. The API Gateway receives these events via the JMS topic and updates its in-memory map.
```

Comment:
Shall we add this content inside the APIM control plane, rather than adding this inside traffic manager.  Because TM will run in data plane, the in memory data are stored in the control plane component.

Reply:
Removed the second paragraph, as that might confuse a user regarding the architecture. We will discuss this in the architecture section when we include it.